spark/datadog_checks/spark/spark.py

```diff
@@ -462,9 +462,13 @@ def _get_spark_app_ids(self, running_apps, tags):
         spark_apps = {}
         version_set = False
         for app_id, (app_name, tracking_url) in iteritems(running_apps):
-            if not version_set:
-                version_set = self._collect_version(tracking_url)
-            response = self._rest_request_to_json(tracking_url, SPARK_APPS_PATH, SPARK_SERVICE_CHECK, tags)
+            try:
+                if not version_set:
+                    version_set = self._collect_version(tracking_url)
+                response = self._rest_request_to_json(tracking_url, SPARK_APPS_PATH, SPARK_SERVICE_CHECK, tags)
+            except RequestException as e:
+                self.log.warning("Exception happened when fetching app ids for %s: %s", tracking_url, e)
+                continue
```

Comment:
Could you add a test to covert this case?

Reply:
👍